### ISSUE TITLE
Add missing Denite configuration to readme

### ISSUE THREAD

**Issue by ryanoasis** [ryanoasis/vim-devicons#205]
#### Required Info

- [x] I have searched the [issues](https://github.com/ryanoasis/vim-devicons/issues) for my issue and found nothing related and/or helpful
- [x] I have read or at least glanced at the [FAQ](https://github.com/ryanoasis/vim-devicons#faq--troubleshooting)
- [x] I have read or at least glanced at the [Wiki](https://github.com/ryanoasis/vim-devicons/wiki)

- [x] What **version** of vim are you using? N/A
  - hint: `vim --version`
- [x] What **version** of vim-devicons are you using? `0.9.1`
  - hint: `:echo webdevicons#version()`
- [x] Are you using vim from the terminal or a GUI vim? N/A
- [x] Are you using Mac, Linux or Windows? N/A

#### Implementation:

Add a Denite section just like we have for unite:

```vim
" adding the custom source to denite 
let g:webdevicons_enable_denite = 1
```

see for example: https://github.com/ryanoasis/vim-devicons/blame/master/readme.md#L308-L311

**Comment by mohamadLy** [comment]
I can take it, if it's ok with you.

**Comment by ryanoasis** [comment]
@mohamadLy Thanks! sure it's all yours :smile:

**Comment by mohamadLy** [comment]
@ryanoasis thanks for giving the chance. I've opened a PR for it. Any feedback is welcome as this is my first time.

**Comment by ryanoasis** [comment]
Closed/fixed via PR #217. Thanks again :+1:
